### ISSUE TITLE
Anomalies when using `--apply-fixes` or Formatting code actions for `@label`

### ISSUE THREAD

**Issue by sebastiaanspeck** [StevenLooman/sonar-magik#386]
`_loop@label` --> `_loop @label`, 
`_continue@label` --> `_continue @label`, 
`_block@label` --> `_block @label`,
`_leave@label` --> `_leave @label`. 

For `_proc@label` it works (kinda) fine. After the `@` an optional space is allowed, but this is raised as an formatting issue.

**Comment by StevenLooman** [comment]
You mean that you don't agree with formatting rules, instead of it being anomalies.

**Comment by sebastiaanspeck** [comment]
I disagree with the fact that `_proc@label` is not being formatted to something else, but other places that can have `@label` are being formatted :)

**Comment by StevenLooman** [comment]
When building this I assumed the label of the `_proc` had no semantic meaning apart from only giving the name of the procedure. This name is used in tracebacks etc. I still believe this is the case. That does not mean I'm not willing to change the formatting though.

There are some snippets to test if it is true that the label of a proc only influeces the name of the procedure, and does not have any meaning in the (possible) control flow alteration:
```magik
Magik> _proc@test() _leave @test _endproc()
unset
Magik> _proc@test() _leave @test _with 10 _endproc()
10
Magik> _proc@test() _leave @other _with 10 _endproc()
10
Magik> _proc@test() _loop _leave @test _with 10 _endloop _endproc()
10

# Here the first `@test` is interpreted as a `sw:global_variable`, as a statement. It does not have any meaning regarding control flow changes.
Magik> _method object.b@test; show(:show_test); _leave @test; show(:show_test) _endmethod
method b in object
Magik> object.b
:show_test
Magik> _method object.b@test; show(:show_test); _leave @test _with :test; show(:show_test) _endmethod
method b in object
Magik> object.b
:show_test
:test

# Here the compiler does not complain about the missing @other label definition.
Magik> _method object.b@test; show(:show_test); _leave @other_with :test; show(:show_test) _endmethod
method b in object
Magik> object.b
:show_test
:test

# Parser does not allow a label definition in/near the method definition.
Magik> _method object.b@test() show(:test); _leave @test; show(:test) _endmethod
**** Error (parser_error): on line 1
_method object.b@test(_optional arg) show(:test); _leave @test; show(:test) _endmethod
                     ^
Expected message send
```

So the label in the proc definition appears to have no significant meaning apart from just being the procedure name.
